pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -191,6 +276,8 @@ type ClusterServiceVersionSpec struct {
 	CustomResourceDefinitions CustomResourceDefinitions `json:"customresourcedefinitions,omitempty"`
 	APIServiceDefinitions     APIServiceDefinitions     `json:"apiservicedefinitions,omitempty"`
 	// +listType=set
+	WebhookDefinitions []WebhookDescription `json:"webhookdefinitions,omitempty"`
```

Comment:
typo: the slice should be []WebhookDefinitions ?

Reply:
See [this comment](https://github.com/operator-framework/operator-lifecycle-manager/pull/1436#discussion_r408147800)